### ISSUE TITLE
Bug with non-lowercase attribute names

### ISSUE THREAD

**Issue by jakearchibald** [WebReflection/hyperHTML#118]
Demo: https://cdn.rawgit.com/jakearchibald/ede7f03929adae492cc9368d4277940b/raw/7fe7e87d56a3f1078439dee607e5559a67e0e2e7/index.html

```js
const output = document.querySelector('.output');
hyperHTML.bind(output, 'svg')`<svg viewBox=${'0 0 50 50'}></svg>`;
```

Throws: Uncaught TypeError: Cannot read property 'nodeType' of undefined.

**Comment by jakearchibald** [comment]
This seems to be because the name in the `NamedNodeMap` is `viewBox`, but hyperHTML looks for the `.toLowerCase` value and ends up with undefined.

https://github.com/WebReflection/hyperHTML/blob/master/index.js#L439

Maybe this should be:

```js
node.attributes[name.toLowerCase()] || node.attributes[name]
```

**Comment by WebReflection** [comment]
thanks @jakearchibald , when you try to code-cover jsdom shenanigans and you forget to test valid cases with camelCase attributes.

Fixed in `v1.10.1`, the check is usually just `node.attributes[name]`, only old browsers and jsdom would fail with non lowercased names :man_shrugging:
